apollo-portal/src/main/resources/static/styles/common-style.css

```diff
@@ -546,6 +546,12 @@ table th {
 
 ::-webkit-scrollbar-thumb:vertical:hover {
     background: rgba(255, 255, 255, 0);
+}*/
+
+.namespace-view-table-scrollbar {
```

Comment:
we already have the `Expand/Collapse` button for every namespace, is it necessary to limit its height?

Reply:
Because it is troublesome to display and view the configuration under a lot of configurations.

For example, there is a namespace with a configuration of 1000 or more. When I move up and down, I sometimes scroll to a very deep place, and then go back and find that the shrink button is far away. If the height is fixed at this time, finding the shrink button will greatly reduce my scrolling operation at this time.

Thank you for your time。

Reply:
Thanks for the feedback. However I don't quite favor the idea to add the scrollbar for configuration items. Wouldn't it be better to add pagination for very long namespaces?

Reply:
Maybe adding a scrollbar is good for our users，Pagination is also a good idea，
I wonder if your future plan for this page long namespaces is about this optimization?

Reply:
Yes, pagination is in the to-do list #1261

Reply:
Maybe i can try add pagination in my company ENV，I can do something for community，I can contribute to the community。

Reply:
This sounds great!